internal/uvm/virtual_device.go

```diff
@@ -0,0 +1,47 @@
+package uvm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/Microsoft/hcsshim/internal/guestrequest"
+	"github.com/Microsoft/hcsshim/internal/requesttype"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+const (
+	resourcePathVPCIFmt = "VirtualMachine/Devices/VirtualPci/%s"
```

Comment:
Put this in resourcepaths.go

Reply:
Did you forget to `git add` resourcepaths.go?

Reply:
this should be updated.